### ISSUE TITLE
Item Stats: Sanfew serum and Relicym's balm missing tooltip info

### ISSUE THREAD

**Issue by F-Lambda** [runelite/runelite#15910]
### Type

Incorrect behavior

### Operating System

Windows

### OS Version

Windows 11

### Bug description

Sanfew serum and Relicym's balm do not provide info on all of their benefits. Sanfew serum only displays the amount of prayer restored, and not the poison and disease immunity durations. Relicym's balm displays no information at all.

### Screenshots or videos

![image](https://user-images.githubusercontent.com/21232805/201400402-7b4c09ae-68d6-4a29-aaca-e91b8d3449b4.png)

![image](https://user-images.githubusercontent.com/21232805/201400449-2719b6cc-2852-4ceb-b21a-43f787c68ea6.png)

### RuneLite version

![image](https://user-images.githubusercontent.com/21232805/201400330-24f82627-28fd-4579-bec8-7fc4a5b25629.png)

**Comment by Nightfirecat** [comment]
If I recall correctly, Sanfew serums (and super restores) show only the prayer restore if nothing else is drained--try draining your other stats and hovering it to see how it behaves then.
Sanfew has antipoison effects which we could add a tooltip for. It also has a lasting anti-disease effect which doesn't exist on any other item, so we could look into possibly adding a tooltip/timer for that, though I think that's not a feature most users care about.
As for Relicym's balm, what tooltip would it show if we added one? It doesn't have any lasting effects--it only cures disease once, and provides no duration of disease immunity.

**Comment by F-Lambda** [comment]
Checked real quick using dragon battleaxe spec, Sanfew serum correctly shows restoring drained stats (or at least drained combat stats):
![image](https://user-images.githubusercontent.com/21232805/201402588-31d91968-5cc5-4eab-8206-d08199ae142f.png)

Relicym's balm unfortunately isn't very well documented. The wiki pages for disease and the potion just say that it "Drinking one or two doses reduces the effect, while four cures most disease altogether. " Would probably need a little experimentation to figure out the numbers.

**Comment by Nightfirecat** [comment]
Is there any kind of disease indicator (amount, or immunity to it) in the enhanced client toolbar on steam/mobile? It's possible, though unlikely I think, there are client vars which indicate that. If not, it's not very likely we would be able to display meaningful information about it.

**Comment by F-Lambda** [comment]
Nope, the only indicator is the disease hitsplat when it procs. Though, I'm unsure if mobile shows poison and venom, either.

**Comment by Nightfirecat** [comment]
There is a var sent to the client indicating the level and duration of poison protection the player has, see: [`VarPlayer#POISON`](https://github.com/runelite/runelite/blob/735bca1d5198a39bcd946d96df99449c06062a17/runelite-api/src/main/java/net/runelite/api/VarPlayer.java#L71-L77) for info on that var, and [`PotionDuration`](https://github.com/runelite/runelite/blob/735bca1d5198a39bcd946d96df99449c06062a17/runelite-client/src/main/java/net/runelite/client/plugins/itemstats/potions/PotionDuration.java) for definitions of potion durations (which is where antipoison/anti-dragon potions get their duration tooltips from)
